src/main/java/picard/sam/HitsForInsert.java

```diff
@@ -217,16 +219,18 @@ public void coordinateByHitIndex() {
                 secondOfPair.add(i, null);
             }
         }
+    }
 
-        // Now renumber any correlated alignments, and remove hit index if no correlated read.
```

Comment:
Thank you for breaking this out and cleaning up hi, and so on.

Reply:
Thank you for your feedback=)